functions/create-or-update-record/1.1/function.json

```diff
@@ -0,0 +1,86 @@
+{
+  "description": "Create or update a record for a given model",
+  "label": "Create or Update Record",
+  "category": "CRUD",
+  "icon": {
+    "name": "DataJumpUpIcon",
+    "color": "Green"
+  },
+  "options": [
+    {
+      "meta": {
+        "type": "Model",
+        "validations": {
+          "required": true
+        }
+      },
+      "name": "model",
+      "label": "Model",
+      "info": "The model you want to create or update a record for."
+    },
+    {
+      "meta": {
+        "type": "PropertyMap",
+        "model": "model"
+      },
+      "configuration": {
+        "dependsOn": [
+          {
+            "option": "model",
+            "action": "CLEAR"
+          }
+        ]
+      },
+      "name": "mapping",
+      "label": "Value Mapping",
+      "info": "Mapping of values to properties for the created or updated record. The properties are based on the model selected."
+    },
+    {
+      "meta": {
+        "allowedKinds": ["INTEGER", "EMAIL_ADDRESS", "TEXT", "STRING", "DATE"],
```

Comment:
maybe extend the list to include all 'string' properties?
```json
[
  "DATE_TIME",
  "DATE",
  "EMAIL_ADDRESS",
  "INTEGER",
  "LIST",
  "STRING",
  "TEXT",
  "URL"
]
```

Reply:
"LIST" could indeed be another useful addition, but for the other ones I don't see a valid use case at the moment.